google/ggrc-core
# GGRC-4662/GGRC-4973 Remove person info pop-up

# Issue description

While hovering over person's email as Default Assignees in Assessment template Info panel, info popup is out off screen

# Steps to test the changes

1. Open any audit page
2. Create Assessment template with any Default Assignee
3. Expand Assessment template info panel
4. Hover over person's email

**Actual Result:** While hovering over person's email as Default Assignees in Assessment template Info panel, info popup is out off screen
**Expected Result:** Remove info pop-up.

# Solution description

Remove unneeded html, js code and css styles.

# Sanity checklist




# PR Review checklist
